core/src/main/scala/org/apache/spark/Logging.scala

```diff
@@ -36,7 +36,7 @@ import org.apache.spark.util.Utils
 trait Logging {
   // Make the log field transient so that objects with Logging can
   // be serialized and used on another machine
-  @transient private var log_ : Logger = null
+  @transient private var _log: Logger = null
```

Comment:
Why don't we reset this change and just turn the rule off here-- otherwise the huge mima exclusion is pretty annoying.